src/prompts/choices/eslint.js

```diff
@@ -0,0 +1,12 @@
+export default [
+  {
+    name: 'js/base',
+    hint: 'Base rules for JavaScript',
+    dependencies: ['eslint-config-airbnb-base@15.0.0']
```

Comment:
It bothers me very much that we have to define the dependencies specific to custom configurations twice: once in the js files and also in the `devDependencies` of `package.json`.

I understand the reason here, it is to not to install peer dependencies for consumers (CLI handles that) while allowing the developers to have all the dependencies installed. I unfortunately cannot think an alternative way at this time but just want to leave this note.

Reply:
I'm going to think about it. Maybe a more centralized approach to avoid having them so scattered in different files. 

They are in fact different set of dependencies, like you point out, `devDependencies` are the ones that this project uses and the ones here are the consumer's dependencies, the ones that get installed in their project when they execute e.g. `code-config eslint` or `npx @maarkllc/code-config eslint`

Reply:
I added dependencies inside each configuration instead of having them here. We still need the distinction between project dependencies and consumer dependencies, so I created `info.json` in each configuration to handle each configuration specific information.